packager/util/config.go

```diff
@@ -24,13 +24,17 @@ import (
 )
 
 type Config struct {
-	LocalOnly bool
-	Port      int
-	CertFile  string // This must be the full certificate chain.
-	KeyFile   string // Just for the first cert, obviously.
-	OCSPCache string
+	LocalOnly	bool
+	Port		int
+	CertFile	string // This must be the full certificate chain.
+	KeyFile		string // Just for the first cert, obviously.
+	NewCertFile	string // The new full certificate chain replacing the expired one.
+	NewKeyFile	string // For the first cert in NewCertFile.
+	AutoRenewCert	bool   // Should we auto-renew cert? Defaults to false.
```

Comment:
(opt) Could we simplify? Auto-renew if and only if there's a non-empty AccountURL?

Reply:
That sounds reasonable.  I will adjust the comments to reflect that decision.